scripts/ceph-windows/run_tests

```diff
@@ -64,4 +64,6 @@ SSH_TIMEOUT=1h ssh_exec powershell.exe /workspace/repos/ceph-win32-tests/test_ho
 ssh_exec curl.exe -s -L -o /workspace/test_rbd_wnbd.py https://raw.githubusercontent.com/ceph/ceph/main/qa/workunits/windows/test_rbd_wnbd.py
 SSH_TIMEOUT=5m ssh_exec python.exe /workspace/test_rbd_wnbd.py --test-name RbdTest --iterations 100
 SSH_TIMEOUT=5m ssh_exec python.exe /workspace/test_rbd_wnbd.py --test-name RbdFioTest --iterations 100
-SSH_TIMEOUT=5m ssh_exec python.exe /workspace/test_rbd_wnbd.py --test-name RbdStampTest --iterations 100
+# We're temporarily disabling this test, see the following for more details:
+#   https://github.com/ceph/ceph/pull/48929
+# SSH_TIMEOUT=5m ssh_exec python.exe /workspace/test_rbd_wnbd.py --test-name RbdStampTest --iterations 100
```

Comment:
This begs the question: what `RbdTest` and `RbdFioTest` do?  `RbdTest` seems to test creating, mapping, unmapping and removing an RBD image -- which makes sense.  `RbdFioTest` seems to boil down to a bunch of reads on an unpopulated (freshly created) image -- which means that it doesn't actually exercise the I/O path.  Was that intended?